kolibri/utils/server.py

```diff
@@ -115,8 +119,9 @@ def start_command(cls):
         thread.start()
 
     def run(self):
-        # Try to do the vacuum every 3 hours
-        call_command("vacuumsqlite", interval=10)
+        with lock:
+            # Try to do the vacuum every 3 hours
```

Comment:
Wondering if this comment makes sense when it says `interval=10` ?

Reply:
planning to change that according to the tagged issue